src/Config.php

```diff
@@ -41,18 +42,19 @@ public static function createContainer($input = null)
         }
 
         // Set up our dependency injection container.
-        $container = new ContainerBuilder();
-        $container->register('logStyler', 'Robo\Log\RoboLogStyle');
-        $container->set('input', $input);
-        $container
-            ->register('output', 'Symfony\Component\Console\Output\ConsoleOutput');
-        $container
-            ->register('logger', 'Robo\Log\RoboLogger')
-            ->addArgument(new Reference('output'))
-            ->addMethodCall('setLogOutputStyler', array(new Reference('logStyler')));
-        $container
-            ->register('resultPrinter', 'Robo\Log\ResultPrinter')
-            ->addArgument(new Reference('logger'));
+        $container = new Container();
+
+        $container->add('input', $input);
+        $container->share('output', 'Symfony\Component\Console\Output\ConsoleOutput');
```

Comment:
let's use `::class` definitions instead of strings, as we are not going to support PHP 5.4